### DISCUSSION TITLE
Consistency with Linux desktop.

### DISCUSSION THREAD

**Discussion by Noob-linux-user** [KRTirtho/spotube#143] [Ideas]
Can we get native header and window bar for linux? I mean I am being picky But it will look coherent, just the window bar will do also if u can make it more gtk it will be great but i guess it will need lot's of work.

Still If you can make the 1st one it will be dope

**Comment by KRTirtho** [comment]
Sorry to say. But this application has to cover 4 platform & if I make consistent only with gtk libadwaita in other platforms it will look out of place. Although libadwaita is completely doable but flutter still doesn't provide platform adaptive widgets except Cupertino for iOS & Material for Android

**Reply by Noob-linux-user** [reply]
Oh I see. I don't know much about how the software stuff. but recently I saw the telegram app. It look out of the place at first but there was an option called use system window frame. it's not much just change the header bar, and make the border more like a system app. Anyways still a nice app :). Have a good day
